### DISCUSSION TITLE
Suggestions for "Install Software"

### DISCUSSION THREAD

**Discussion by SamHills** [99natmar99/Windows-11-Fixer#7] [Ideas]
Please add these programs to the "Install Software" page:
Browsers:
  Brave
  Ungoogled Chromium

Office:
  Microsoft Office 2021

Gaming:
  Origin

Editing:
  Irfanview

Other
  7-Zip (instead of WinRAR)

**Comment by 99natmar99** [comment]
Thank you for your feedback.

For all programs listed on the install page, they are installed through a service called winget. Therefore I am limited to the programs that the service contains.

In regards to Origins, unfortunately that program is not able to be added. However EA App is the next evolution of Origin and can be used to access the same game library.

Microsoft Office 2021 is another program I am unfortunately unable to add. However Microsoft 365 includes all the latest versions of the Microsoft office suite programs. I am uncertain though as to whether it will allow a 2021 license activation or not if you don't have an office 365 subscription.

7-Zip will be included in the next update to Windows 11 Fixer.

I will look into the other program mentioned and some (or all) of them may be included in the next update to Windows 11 Fixer.

**Reply by SamHills** [reply]
Thanks for your response.  I already have a collection of batch files that I use to configure Windows, but I'm always looking for new things to add to it so I investigated your program to see if it had anything I didn't already have.

I will probably peruse your source code to see how you uninstalled the "hard-to-uninstall" apps.  I'm not really satisfied with the results I'm getting with my code.

I use Chocolatey instead of Winget.  I tried Winget once for a program that was not yet available via Chocolatey but never did get it to work right.   You might want to look into Chocolatey for those apps not available on Winget, such as Origin.

Office 2021 is not yet available on Chocolatey, so I wrote a batch file to do the job.  It downloads Office 2021 from the M$ website and activates it via a KMS server.
